### DISCUSSION TITLE
nanovdb on webgpu

### DISCUSSION THREAD

**Discussion by unicomp21** [AcademySoftwareFoundation/openvdb#1625] [Q&A]
would this make sense?  nanovdb in a browser, running on webgpu?

**Comment by emcfarlane** [comment]
I tried a proof of concept port of NanoVDB to the browser: https://github.com/emcfarlane/webgpu-nanovdb

Theres some limitations that make it awkward working with in WGSL. Lack of 64-bit types, limited type casting. The port is based on the PNanoVDB.h header. See the demo.
